### PR TITLE
Fixed almost all Webhint warnings and added accessibility

### PR DESCRIPTION
**Pull request by Kakalanp**
### List all points checked:

- Page titles : **no issues found.**
> Adding spacing in head for easier readability.
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L12-L17

- Image text alternatives : _**changes below.**_
>Added explicit but short “alt” descriptions inside important images and removed useless descriptions that may be disorienting for people with screen readers.
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L51-L57
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L66
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L72
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L89
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L107
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L124
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L142
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L161
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L191-L212
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L239-L245

- Text headings : **no issues found.**

- Color contrast : _**changes below.**_
>Changed button color to have more contrast between text and background.
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/styles.css#L140

- Resize : _**changes below.**_
>removed `height: 95vh;` from the landing page container (used to be CSS line 40), as it served no propose and caused some visual overlap of text at zoom 400%*
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/styles.css#L39-L40

- Interaction : **no issues found.**

- Moving content : **no issues found.**

- Multimedia : **no issues found.**

- The basic structure of the page : **no issues found.**

### Extras:

-Webhint warnings: 
>Added “noreferrer” property to “rel” inside most “a” tags.
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L51-L57
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/index.html#L239-L245

-Bug prevention:
>Changed `background: #008552;` to `background-color: #008552;` just to prevent it from breaking.
https://github.com/Kakalanp/Portfolio-Joaquin/blob/7f6dc6845eaaed4fc9acddc493c443c90bad35be/styles.css#L158

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,13 +48,13 @@ <h2 class="enphasis">I'm a software developer</h2>
...
+                    <a target="_blank" href="https://github.com/Kakalanp" rel="noopener noreferrer"><img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt="GitHub profile"/></a>
                 </li>
                 <li class="social">
-                    <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="li"/></a>
+                    <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener noreferrer"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="Linkedin profile"/></a>
                 </li>
                 <li class="social">
-                    <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="t"/></a>
+                    <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener noreferrer"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="Twitter profile"/></a>
```
Comment (AleksandarIvezic) [inline index.html:57]: [Required]
- [x] Please make sure that you added links to your changes into your report as it was required in project instructions.

[Optional] 

- [ ] You can revert those commits and make changes on a new branch. Then you can add link to changes from that branch to your report on this branch.

File: index.html [hunk 2]
```diff
@@ -234,18 +236,19 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
...
                 </li>
```
Comment (AleksandarIvezic) [inline index.html:246]: [Required]
- [x] Same like for all other changes - please make sure that you added links to your changes into your report as it was required.

[Optional] 

- [ ] You can revert those commits and make changes on a new branch. Then you can add link to changes from that branch to your report on this branch.

**Review by AleksandarIvezic — CHANGES_REQUESTED** [review]
## Hi @Kakalanp @mauriciomonta  👋🏼 , 

### STATUS: Changes requested! ♻️ ♻️ ♻️ 

You did a solid job so far! 👏🏼 

However, there are some issues that you still need to work on to go to the next project but you are almost there!
Please pay attention about instructions how to make changes on your code. It should not be done directly but by opening a new branch and linking the changes you did to your report. 😉 


image


## Required Changes ♻️
_Check the comments under the review._



- [ ]  Please make sure that you include links to your changes:

- If you needed to change something, [link to the corresponding line of code in your pull request changes](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148).
- Instead of this list you created
image
you just need to add link to the line in your code you changed near the point it refers to.
For example: 
- Page titles: [Change 1](https://github.com/Kakalanp/Portfolio-Joaquin/blob/921f9356d310899390bce290196e9b5ffaa5f3ed/index.html#L14-L16)

- [x] Please make sure that you have only 1 `<h1>` tag on your page and that it stands on the top of your page - [headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings);
image
image

- [ ] Please make sure that the page has a minimum contrast ratio of at least 4.5:1 for normal-size text.You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help checking the color contrast of the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
> * Please make sure that the page has a minimum contrast ratio of at least 4.5:1 for normal-size text.You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help checking the color contrast of the project.

About this part, I'm a little confused with the color change, I'm using the colors from the Figma design, should I change those to whatever contrasts no matter the color nor the Figma-template-fidelity?

**Comment by AleksandarIvezic** [comment]
Hey @Kakalanp , the only thing you should do there is to change the shade of your buttons. For example instead of `.button { background: #36b37f }` you can use some darker shade like this one `#2b8e65`. If you are using Firefox browser I would recommend you to install [this addon](https://addons.mozilla.org/en-US/firefox/addon/wcag-contrast-checker/) so in combination with developer tools you can find the optimal colors combination.

**Comment by Kakalanp** [comment]
> the only thing you should do there is to change the shade of your buttons

Thank you for the clarification! 🔍 

I just updated everything you pointed out, and I'm about to submit my project for another review, many thanks for your help! 😁

**Comment by AleksandarIvezic** [comment]
> > the only thing you should do there is to change the shade of your buttons
> 
> Thank you for the clarification! 🔍
> 
> I just updated everything you pointed out, and I'm about to submit my project for another review, many thanks for your help! 😁

You are very welcome. I am glad to help. Please, don't forget to add links to corresponding changes before you submit for another review. 😉

**Comment by Kakalanp** [comment]
I just did, I'm not sure if it's too much, but it surely passes the requirements.

**Review by Meltrust — APPROVED** [review]
Hi @Kakalanp, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**